### PR TITLE
Tweak Gruntfile.js

### PR DESCRIPTION
**Pull request by XhmikosR**
Trying to improve the current situation :)

~~Only thing left is to copy only a specific theme's files and not all.~~ **EDIT**: done!

### CODE REVIEW

**Comment by XhmikosR** [comment]
This is ready but I'd like to get #1002 merged and test it a bit yourself if you could @thomaspark.

**Comment by thomaspark** [comment]
#1002 is merged. I tested it out and it's working perfectly on my end.

File: Gruntfile.js [hunk 1]
```diff
@@ -108,99 +125,100 @@ module.exports = function (grunt) {
...
   });
 
-  grunt.registerTask('compress', 'minify a built css file', function(fileSrc, fileDst) {
-    var distPost = {src: fileSrc, dest: fileDst};
-    grunt.config.set('cssmin.dist', distPost);
-    grunt.log.writeln('Minifying file ' + fileSrc);
-    grunt.task.run(['cssmin:dist']);
-  });
-
   grunt.registerMultiTask('swatch', 'build a theme', function() {
```
Comment (XhmikosR) [inline Gruntfile.js:206]: I don't quite get this task. Is it even used?

Comment (thomaspark) [reply Gruntfile.js]: It was used once upon the time. Might have been v2!

Comment (XhmikosR) [reply Gruntfile.js]: We should remove it then, since it's confusing :)

**Pull request merged**